src/middlewares/tripMiddleware.js

```diff
@@ -3,7 +3,14 @@ import { respondWithWarning } from '../helpers/responseHandler';
 import statusCode from '../helpers/statusCode';
 
 export const verifyTrip = async (req, res, next) => {
-  const trip = await findTripById(req.params.tripId);
+  
```

Comment:
Kindly stick to `tripId` as the parameter for trip id and user `Number(req.params.tripId)` instead of `parseInt(req.params.tripId)`.